src/dma/collector/workflows/readiness_check/_postgres/main.py

```diff
@@ -172,15 +178,21 @@ def _check_pglogical_installed(self) -> None:
                     c.db_variant,
                     rule_code,
                     "PASS",
-                    "`pglogical` is installed on the database",
+                    "`pglogical` is installed on the database.",
                 )
 
     def _check_wal_level(self) -> None:
         rule_code = "WAL_LEVEL"
         if self.db_version is None:
```

Comment:
Can we move this check to the execute() method above so that we don't need to repeat this in each new check?

Reply:
It's really more of an assertion on the type.  It's going to think it could be optionally null here when in reality it's not possible for it to be null at this point of the program.

Reply:
I've just overridden then type signature so that we no longer need this to satisfy the linter.